### ISSUE TITLE
Unlisted toots shouldn't show up on the user timeline

### ISSUE THREAD

**Issue by ticki** [tootsuite/mastodon#3799]
Currently, the "unlist" mode for toots only removes it from the feed of others. It doesn't stop cluttering the timeline of an user.

On Twitter, the timeline is divided into a "tweets" and a "tweets & replies" part. The latter also shows replies, which under Mastodon would correspond to "unlisted" toots.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by Kimik0** [comment]
"Unlisted" means the option you select from the visibility menu before tooting. "Reply" means a toot is linked to another toot that it replies to. Both unlisted toots and replies are not shown on the public timelines, but otherwise they're different things altogether.
Your own toots show on your own timeline. If you don't want to see them, why make them in the first place?

**Comment by ticki** [comment]
@Kimik0 The exact problem is that the toot is listed on my timeline, despite being in unlisted mode (i.e. I want the person I'm replying to and the other people reading the thread to see it). This, like Twitter does, shouldn't show up on my timeline, yet it does.

**Comment by ticki** [comment]
With "timeline" I mean the user page (like this: https://mastodon.social/@ticki), maybe I misapply the terminology.

**Comment by unarist** [comment]
If you want to separate "Toots" and "Toots & Replies" like Twitter, it looks like related to #3764.
and if you want to remove mentions & replies completely from your timeline... use "private" or "direct" visibility.

Also you should read User's Guide if you haven't read yet:

https://github.com/tootsuite/documentation/blob/master/Using-Mastodon/User-guide.md#toot-privacy

**Comment by joyeusenoelle** [comment]
@unarist is correct: "Unlisted" means **only** that it doesn't appear on the public timeline. If you want a post to only appear to the people included in the conversation, use the "Direct" visibility setting.

**Comment by ticki** [comment]
@joyeusenoelle Thing is though that I want people who come across the thread to be able to read the toots, not just the users I'm replying to.

**Comment by Cassolotl** [comment]
> Thing is though that I want people who come across the thread to be able to read the toots, not just the users I'm replying to.

We now have this in profiles in the web UI and on individual profile pages: split into toots, toots and replies, and media.

![screen shot 2018-04-05 at 10 07 48](https://user-images.githubusercontent.com/6607512/38357001-3f4debd8-38b9-11e8-8305-fc291a5ab0b3.png)

"Toots" includes original posts and boosts.

Unlisted posts still appear on one's profile, though.

Does this meet your needs, @ticki?

**Comment by Gargron** [comment]
I don't know why I just got a notification for this thread today, but it's a good thing I did:

>On Twitter, the timeline is divided into a "tweets" and a "tweets & replies" part.

We now have this too!

Generally, the "unlisted" option was requested by people who didn't want to appear on the "federated" and "local" and any hashtag timelines, i.e. not attract non-followers, but otherwise shareable and viewable. It corresponds roughly to the YouTube "unlisted" setting for videos which it got its name from. In so far as it's purposefully shareable and viewable by anybody, I don't see a reason why it shouldn't be on the user profile.

I hope nobody has a problem with this issue being closed.
